_sass/_md.scss

```diff
@@ -1,4 +1,20 @@
 @media #{$md} {
+	.searchfield {
+		top: 82px;
+        width: 85vw;
+        max-width: 1100px;
```

Comment:
Where'd 1100px come from? Why not either full width or line up with the right edge of the content?